devfiles/isce3/devfile/meta.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 displayName: "ISCE3"
-description: "ISCE3 workspace version: 3.1.5"
+description: "ISCE3 MAAP workspace"
```

Comment:
`ISCE3 MAAP workspace version: 4.0.0`

Can we keep the version number in the name until we have another way for us to know what version we are running from within the workspace.